src/Slider.js

```diff
@@ -165,10 +198,17 @@ var Slider = React.createClass({
     });
   },
   componentWillReceiveProps: function(nextProps) {
-    var oldValue = this.props.value;
-    var newValue = nextProps.value;
+    var props    = this.props,
+        oldValue = props.value,
+        newValue = nextProps.value;
+
```

Comment:
The pattern of development that I've used is geared towards better magnification & faster processing by the interpreter.